### PR TITLE
Upgrade containerd and image-spec for the vulnerabilities

### PR DESCRIPTION
**Pull request by mopp**
## Background

We have used migrate in our applications and configured Dependabot alerts by GitHub.
It notified us that containerd and image-spec had vulnerabilities. But we don't use them directly.
Then, I created this PR to upgrade.

I am not sure how migrate use them 🙏

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,8 +9,9 @@ require (
 	github.com/apache/arrow/go/arrow v0.0.0-20211013220434-5962184e7a30 // indirect
 	github.com/aws/aws-sdk-go v1.17.7
 	github.com/aws/aws-sdk-go-v2/feature/s3/manager v1.5.4 // indirect
-	github.com/cenkalti/backoff/v4 v4.0.2
+	github.com/cenkalti/backoff/v4 v4.1.1
 	github.com/cockroachdb/cockroach-go/v2 v2.1.1
+	github.com/containerd/containerd v1.5.8 // indirect
```
Comment (mopp) [inline go.mod:14]: See https://github.com/containerd/containerd/releases/tag/v1.5.8

File: go.mod [hunk 2]
```diff
@@ -40,6 +41,7 @@ require (
 	github.com/mutecomm/go-sqlcipher/v4 v4.4.0
 	github.com/nakagami/firebirdsql v0.0.0-20190310045651-3c02a58cfed8
 	github.com/neo4j/neo4j-go-driver v1.8.1-0.20200803113522-b626aa943eba
+	github.com/opencontainers/image-spec v1.0.2 // indirect
```
Comment (mopp) [inline go.mod:44]: See https://github.com/opencontainers/image-spec/releases/tag/v1.0.2

File: go.mod [hunk 3]
```diff
@@ -9,8 +9,9 @@ require (
 	github.com/apache/arrow/go/arrow v0.0.0-20211013220434-5962184e7a30 // indirect
 	github.com/aws/aws-sdk-go v1.17.7
 	github.com/aws/aws-sdk-go-v2/feature/s3/manager v1.5.4 // indirect
-	github.com/cenkalti/backoff/v4 v4.0.2
+	github.com/cenkalti/backoff/v4 v4.1.1
```
Comment (mopp) [inline go.mod:12]: See https://github.com/cenkalti/backoff/compare/v4.0.2...v4.1.1

**Comment by dhui** [comment]
My guess is that these dependencies are via [dktest](https://github.com/dhui/dktest/issues/10) but there may be other packages that depend on these packages as well. I'm not too concerned about these from a security perspective since these packages should only be used when tests are running, so you shouldn't be vulnerable via `migrate` unless you're running the `migrate` tests. Nonetheless, we'll make sure this is fixed in the next release.

**Comment by dhui** [comment]
Could you try updating dktest to [v0.3.8](https://github.com/dhui/dktest/releases/tag/v0.3.8) and run `go mod tidy` to see if that fixes the issue?

**Comment by mopp** [comment]
Thanks for your comment.
Ah, I misunderstood. These libraries are also indirect dependencies from `migrate`.
Please check the commit 
https://github.com/golang-migrate/migrate/pull/662/commits/9975d487b050ab6ad53bd7cf819d18bd98ad5edf

Yes, I know that `migrate` uses them in the test only. But the Dependabot alerts by GitHub does not regard that unfortunately and just send us vulnerability alert 😢 So the reason why I sent this PR is just to suppress the alert in our repository.

Thanks.

File: go.mod [hunk 4]
```diff
@@ -9,11 +9,11 @@ require (
...
 	github.com/cockroachdb/cockroach-go/v2 v2.1.1
 	github.com/cznic/mathutil v0.0.0-20180504122225-ca4c9f2c1369 // indirect
 	github.com/denisenkom/go-mssqldb v0.10.0
-	github.com/dhui/dktest v0.3.7
+	github.com/dhui/dktest v0.3.8
```
Comment (mopp) [inline go.mod:16]: See https://github.com/dhui/dktest/releases/tag/v0.3.8

**Review by dhui — APPROVED** [review]

**Comment by xrn** [comment]
@dhui any chance for a new release with these changes?

**Comment by Tarang** [comment]
Any chance this can be put into release?

**Comment by xrn** [comment]
@dhui ?

**Comment by dhui** [comment]
I just updated dktest to v0.3.10 in migrate (master branch) which should fix known security issues and appease the security vulnerability scanners. If you'd also like to quiet your vulnerability scanners, use the master branch until the next release is cut. Although the master branch is not stable, any issues caused by recent changes (e.g. PR merges) will be promptly addressed.

**Pull request merged**